robotbase/src/main/java/com/team2357/frc2022/commands/IntakeSequenceCommandGroup.java

```diff
@@ -0,0 +1,30 @@
+package com.team2357.frc2022.commands;
+
+import com.team2357.frc2022.Constants;
+import com.team2357.frc2022.subsystems.IntakeSubsystem;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+
+
+public class IntakeSequenceCommandGroup extends SequentialCommandGroup{
+private IntakeSubsystem m_intakeSub;
+
+public IntakeSequenceCommandGroup(IntakeSubsystem intakeSub){
+    m_intakeSub = intakeSub; 
+    addRequirements(m_intakeSub);
+
+if (m_intakeSub.getPivot() == Value.kReverse){
+    addCommands(new IntakeSetPivotCommand(m_intakeSub, Value.kForward), new WaitCommand(Constants.INTAKE.PIVOT_WAIT_SECONDS), new IntakeRollerCommand(m_intakeSub, Constants.INTAKE.FORWARD_SPEED));
```

Comment:
Same thing on the `IntakeSetPivotCommand`. It's really just an extension of the subsystem. I'd really like to have two separate commands like `IntakeDeployCommand` and `IntakeStowCommand` because we then can adjust timing and values for each operation separately.

Same thing for `IntakeRollerCommand`. I'd rather have an `IntakeRollerActive` which takes no parameters, only uses constants.